.expeditor/buildkite/cypress.sh

```diff
@@ -18,5 +18,5 @@ do
   export CYPRESS_RECORD_KEY="$CYPRESS_RECORD_KEY"
 
   npm install # get dependencies defined in e2e/package.json
-  cypress run --record
+  npm run cypress:run --record
```

Comment:
That's a thing? Huh. I wouldn't have thought that this would work 🤔

Reply:
lemme double-check, pretty sure it should work

Reply:
oo good call that didn't work like i hoped 😅

Reply:
Sorry for spoiling the post-merge surprise 🎉

Reply:
now that we're saving the failure screenshots and videos as buildkite artifacts, i wonder what we're really getting out of recording to the cypress dashboard anyway 💭

Reply:
my fly-by two cents is that we should drop the recording to cypress dashboard. only a few of us have access anyways and if we're already uploading things as bk artifacts, then, as you suggest, i don't think we're getting much out of it

Reply:
Good point -- however, we're **not** uploading artifacts in the deploy/dev test code (the file we're commening on).

Reply:
ooooh, darn -- is there a way to do so?

Reply:
Sure, we do it here: https://github.com/chef/automate/pull/837/files#diff-58a979e454f063e727ec94a03c56253dR22

Reply:
What I have no idea about is: _Where are BK artifacts stored? Do they cause much costs on the long run?_ (It's probably S3; but is it _our_ S3 or BK's S3? (Do they bill for artifacts?))

Reply:
Pinged releng about it: https://chefio.slack.com/archives/C074S2A94/p1562868350187500

Reply:
i had added that 👍

Reply:
i'll keep it recording for now but maybe after we've seen a few deploys save the recordings as expected, i'll go in and drop the recording stuff

Reply:
depending on what releng says